ironfish/src/blockchain/blockchain.ts

```diff
@@ -956,7 +956,7 @@ export class Blockchain {
       if (index < noteCount) {
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
         const oldNote = (await this.notes.get(index, tx))!
-        if (!this.strategy.noteSerde.equals(note, oldNote)) {
+        if (!note.serialize().equals(oldNote.serialize())) {
```

Comment:
Move this to `.equals` function on the note class

Reply:
Yeah I expect that to just be checking the note hashes for equality